src/sonic-config-engine/tests/sample_output/py3/buffers-mellanox2410.json

```diff
@@ -0,0 +1,1163 @@
+
+{
+    "CABLE_LENGTH": {
```

Comment:
What is the diff between this one and buffers-mellanox2700.json?

Reply:
2700 should be single ingress pool. This one should be double ingress pools